frontend/src/app/modules/availability/new-slot/new-availability-page/new-availability-page.component.ts

```diff
@@ -61,11 +62,15 @@ export class NewAvailabilityPageComponent extends BaseComponent {
             generalDetails: general,
             advancedSettings,
             eventDetails: this.newAvailabilityComponent.eventDetailComponent.settings,
-            questions: this.newAvailabilityComponent.questionsComponent.questions,
             schedule: this.newAvailabilityComponent.scheduleComponent.schedule,
             templateSettings: this.newAvailabilityComponent.notificationEmailsComponent.settings,
             teamId: this.currentTeamId,
             hasAdvancedSettings: this.newAvailabilityComponent.generalComponent.addAdvanced,
+            questions: this.newAvailabilityComponent.questionsComponent.questions.map((question) => {
+                question.questionText = removeExcessiveSpaces(question.questionText);
+
+                return question;
+            }),
```

Comment:
```suggestion
            questions: this.newAvailabilityComponent.questionsComponent.questions,
```
The _questions component_ should be responsible for correct processing of user input and presenting it in a usable form.